packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -432,6 +434,7 @@ export class HorizontalScroll extends FoundationElement {
         steps.push(newPosition);
 
         this.move(steps, this.frameTime);
+        this.$emit("scrollend", newPosition);
```

Comment:
I think you'll want to move this inside of the this.move() after it checks if there are any remaining steps. After line 455.

Reply:
O, that makes sense. I originally read it as a recursive function, but I now see it is called within the requestAnimationFrame callback.

Going to drop the newPosition since I don't have a use for it, so I don't want to bother passing that to move.